src/components/ProductFilter.jsx

```diff
@@ -0,0 +1,99 @@
+import "./component-css/productFilter.css";
+
+export const ProductFilter = () => {
+  return (
+    <aside className="products-filter-aside">
+      <div className="filter filter-heading">
+        <h3>Filter</h3>
+        <span>
+          <a href="/" className="filter-clear">
+            Clear
+          </a>
+        </span>
+      </div>
+
+      <div className="filter filter-sort-by">
+        <h3>Sort by</h3>
+        <div className="filter-sort-by-wrapper">
+          <label>
+            <input type="radio" name="sort-by" />
+            Price : High to Low
+          </label>
+        </div>
+        <div className="filter-sort-by-wrapper">
+          <label>
+            <input type="radio" name="sort-by" />
+            Price : Low to High
+          </label>
+        </div>
+      </div>
+
+      <div className="filter filter-category">
+        <h3>Category</h3>
+        <div className="filter-category-wrapper">
+          <label>
+            <input type="checkbox" name="category" />
+            Fiction
+          </label>
+        </div>
+        <div className="filter-category-wrapper">
+          <label>
+            <input type="checkbox" name="category" />
+            Romance
+          </label>
+        </div>
+        <div className="filter-category-wrapper">
+          <label>
+            <input type="checkbox" name="category" />
+            Self Help
+          </label>
+        </div>
+        <div className="filter-category-wrapper">
+          <label>
+            <input type="checkbox" name="category" />
+            Mystery
+          </label>
+        </div>
+      </div>
```

Comment:
suggestion: you can map this from an array

Reply:
Noted. I will take care of this in the subsequent PRs